Author: Steven Ayre <steveayre@gmail.com>
Subject: Disable some failing tests
Last-Update: 2012-06-12

--- a/mysql-test/suite/ndb/t/disabled.def	2012-03-06 18:29:31.000000000 +0000
+++ b/mysql-test/suite/ndb/t/disabled.def	2012-03-06 18:31:55.000000000 +0000
@@ -17,3 +17,6 @@
 ndb_disconnect_ddl        : Bug#31853 flaky testcase...
 
 ndb_dbug_lock : SEAGULL gsl locking order
+
+ndb.ndb_join_pushdown_default : Fails on 3rd pass with memory error
+
