Author: Nicholas Bamber <nicholas@periapt.co.uk>
Subject: certain tests failing
Bug-Debian: http://bugs.debian.org/674267
Last-Update: 2012-05-25
Forwarded: no
--- a/mysql-test/suite/rpl/t/disabled.def
+++ b/mysql-test/suite/rpl/t/disabled.def
@@ -18,3 +18,6 @@
 rpl_empty_master_host : bug#13003111 valgrind warning
 rpl_heartbeat_basic : Fails intermittently on AMD64 buildds http://pad.lv/894146
 rpl_mysql_upgrade: Fails on Debian #64773
+rpl_heartbeat_ssl:     Debian Bug 674267 2012-05-24
+rpl_ssl1: Debian Bug 674267 2012-05-24
+rpl_ssl: Debian Bug 674267 2012-05-24
--- a/mysql-test/t/disabled.def
+++ b/mysql-test/t/disabled.def
@@ -29,6 +29,8 @@
 ssl_compress             : Bug#64773 2012-03-27 Debian local
 ssl                      : Bug#64773 2012-03-27 Debian local
 openssl_1                : Bug#64773 2012-03-27 Debian local
+ssl_cipher               : Debian Bug 674267 2012-05-24
+ssl_connect              : Debian Bug 674267 2012-05-24
 
 ### MCP_DISABLE_FAILING_TESTS start ###
 file_contents            : Bug#12585902 2011-09-27 magnus file_contents.test fails when building from "bzr export"
